glusterd2/transaction/transaction-rpc.proto

```diff
@@ -9,7 +9,6 @@ message TxnStepReq {
 
 message TxnStepResp {
   string Error = 1;
-  bytes Resp = 2; // Resp is JSON encoded TxnCtx
```

Comment:
Please mark the removed field and number as [reserved][1].

[1]: https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Done.